scripts/build.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+
+/**
+ * Dummy build script to satisfy Github Actions build requirement for running unit tests.
+ */
+console.log("done");
```

Comment:
So the validate PR script expects a build step. This repo doesn't have one so I just put a dummy script in place.